tests/AuthenticatedTestCase.php

```
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Tests;
+
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class AuthenticatedTestCase extends WebTestCase
+{
+    public function login(KernelBrowser $client, string $email): void
+    {
+        $userRepository = static::getContainer()->get(UserRepository::class);
+        $user = $userRepository->findOneBy(['email' => $email]);
+
+        if ($user) {
+            $client->loginUser($user);
+        }
```

Maybe we throw an exception if user is not found ? Could be time saving ?